src/Dscp/Web/Server.hs

```diff
@@ -0,0 +1,21 @@
+
+-- | Utils for serving HTTP APIs
+
+module Dscp.Web.Server
```

Comment:
`ServerUtils` maybe then?

Reply:
If I wrote "Methods for serving HTTP APIs", you'd suggest to rename it to `ServerMethods`?
I'd better leave it like that to be more consistent with naming of actual API serving modules (e. g. `Dscp.Educator.Web.Student.Server`, later there will be `Dscp.Witness.Web.Explorer.Server`, etc.)